.github/dependabot.yml

```diff
@@ -4,3 +4,6 @@ updates:
   directory: "/"
   schedule:
       interval: "weekly"
+  commit-message:
+      prefix: ":seedling:"
```

Comment:
nit: does this need a trailing space?

Reply:
That has been my understanding with the `prefix` values. Looks good!

Reply:
Perfect!